pkg/tfvars/openstack/openstack.go

```diff
@@ -41,20 +41,28 @@ type config struct {
 	AdditionalSecurityGroupIDs []string `json:"openstack_master_extra_sg_ids,omitempty"`
 	MachinesSubnet             string   `json:"openstack_machines_subnet_id,omitempty"`
 	MachinesNetwork            string   `json:"openstack_machines_network_id,omitempty"`
+	MasterAvailibilityZones    []string `json:"openstack_master_availibility_zones,omitempty"`
```

Comment:
s/availibility/availability/g :)
Make sure to catch all the occurrences.

Reply:
:(